src/front/scss/layout/_spinner.scss

```diff
@@ -0,0 +1,11 @@
+@import '../../../../node_modules/spinkit/scss/spinners/3-wave';
```

Comment:
does absolute path import not configured in this project?

Reply:
seems like doesn't want to do it in pre-processor

Reply:
Did you tried this?
```css
@import '~foundation-sites/scss/spinners/3-wave';

Reply:
No. But I'll try